app/build.gradle

```diff
@@ -13,12 +13,12 @@ apply plugin: 'android-sdk-manager'
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion 24
```

Comment:
NOt knowing much of anything about Gradle, does this change what we support? Why is it bumped here?

Reply:
```AccessibilityAction.ACTION_SET_PROGRESS``` was introduced in API 24

Reply:
Does this change what we support?

Reply:
No. It just for compilation purposes.

Reply:
Ok. Thanks.